base_layer/wallet/src/config.rs

```diff
@@ -136,7 +136,7 @@ impl Default for WalletConfig {
             base_node_service_config: Default::default(),
             data_dir: PathBuf::from_str("data/wallet").unwrap(),
             db_file: PathBuf::from_str("db/console_wallet.db").unwrap(),
-            db_connection_pool_size: 5, // TODO: get actual default
+            db_connection_pool_size: 16, // Note: Do not reduce this default number
```

Comment:
this is really high.  I was expecting it to be only one or two connections at a time

Reply:
From the start we have always tested with 16; this number slipped in somewhere. Effectively it makes read operation asynchronous.